conda/recipes/libcuvs/recipe.yaml

```
@@ -0,0 +1,408 @@
+        - libraft-headers =${{ minor_version }}
+        - librmm =${{ minor_version }}
+        - nccl ${{ nccl_version }}
+        - if: cuda_major == "11"
+          then:
+            - cuda-profiler-api =${{ cuda11_cuda_profiler_api_host_version }}
+            - libcublas =${{ cuda11_libcublas_host_version }}
+            - libcublas-dev =${{ cuda11_libcublas_host_version }}
+            - libcurand =${{ cuda11_libcurand_host_version }}
+            - libcurand-dev =${{ cuda11_libcurand_host_version }}
+            - libcusolver =${{ cuda11_libcusolver_host_version }}
+            - libcusolver-dev =${{ cuda11_libcusolver_host_version }}
+            - libcusparse =${{ cuda11_libcusparse_host_version }}
+            - libcusparse-dev =${{ cuda11_libcusparse_host_version }}
+          else:
+            - cuda-cudart-dev
+            - cuda-profiler-api
+            - libcublas-dev
+            - libcurand-dev
+            - libcusolver-dev
+            - libcusparse-dev
+      run:
+        - ${{ pin_subpackage("libcuvs", exact=True) }}
+        - ${{ pin_compatible("cuda-version", upper_bound="x", lower_bound="x") }}
+        - nccl
+        - if: cuda_major == "11"
+          then: cudatoolkit
+          else:
+            - cuda-cudart
+            - libcublas
+            - libcurand
+            - libcusolver
+            - libcusparse
+      ignore_run_exports:
+        by_name:
+          - cuda-cudart
+          - cuda-version
+          - libaio
+          - libboost
+          - libcublas
+          - libcurand
+          - libcusolver
+          - libcusparse
+          - librmm
+          - mkl
+          - nccl
+          - if: cuda_major == "11"
+            then: cudatoolkit
+    about:
+      homepage: ${{ load_from_file("python/libcuvs/pyproject.toml").project.urls.Homepage }}
+      license: ${{ load_from_file("python/libcuvs/pyproject.toml").project.license.text }}
+      summary: libcuvs example executables
+
+  - package:
+      name: libcuvs-bench-ann
+      version: ${{ version }}
+    build:
+      script:
+        content: |
+          cmake --install cpp/build --component ann_bench
+
+          if [ "$RAPIDS_CONDA_ARCH" != "aarch64" ]; then
+            # Need this so from the diskann compile, but there isn't a
+            # component associated with it. Probably a better way to do this.
+            cp cpp/build/_deps/diskann-build/src/libdiskann.so $PREFIX/lib/
+          fi
+      string: cuda${{ cuda_major }}_${{ date_string }}_${{ head_rev }}
+      dynamic_linking:
+        overlinking_behavior: "error"
+    requirements:
+      build:
+        - cmake ${{ cmake_version }}
+        - ${{ stdlib("c") }}
+      host:
+        - librmm =${{ minor_version }}
+        - libraft-headers =${{ minor_version }}
+        - nccl ${{ nccl_version }}
+        - ${{ pin_subpackage("libcuvs", exact=True) }}
+        - cuda-version =${{ cuda_version }}
+        - openblas # required by some CPU algos in benchmarks
```

Separate comments in YAML by two spaces.
```suggestion
        - openblas  # required by some CPU algos in benchmarks
```